corehq/apps/locations/tests/test_location_queries.py

```diff
@@ -87,6 +89,11 @@ def test_getitem_with_slice(self):
         locs = SQLLocation.objects.get(name='Suffolk').get_descendants()
         self.assertEqual([x.name for x in locs[:2]], ['Boston'])
 
+    def test_pickle_descendants_query(self):
+        locs = SQLLocation.objects.get(name='Suffolk').get_descendants()
+        # should not raise excepiton
+        pickle.dumps(locs)
```

Comment:
Ah, looks like an ICDS dashboard thing - not sure it's good practice to pass a django queryset as a parameter to celery, but I wouldn't want to go to far down the rabbit hole in custom code.

Reply:
> not sure it's good practice to pass a django queryset as a parameter to celery

I agree, although I don't think CTE queries are the correct place to block that, so in general I think they should support pickling like other querysets. It might make sense to implement something to alert someone when a queryset is pickled.